src/Raven.Client/ServerWide/ExternalReplication.cs

```diff
@@ -13,6 +13,7 @@ public class ExternalReplication : ReplicationNode, IDatabaseTask, IDynamicJsonV
         public string Name;
         public string ConnectionStringName;
         public string MentorNode;
+        public long DelayReplicationFor; 
```

Comment:
I don't like this data type. Delay replication for what? Ticks, seconds, minutes, years?
Better to use TimeSpan here.

Reply:
Seconds would be ok? so we can also test it in reasonable time.

Reply:
Any reasons _not_ to use timespan?